### DISCUSSION TITLE
Issue with Brief Silence Between Songs in AzuraCast

### DISCUSSION THREAD

**Discussion by heathcliff2019** [savonet/liquidsoap#4190] [Q&A]
Hello,
I am experiencing a problem with my radio station using AzuraCast. There is a brief silence of about 3 seconds at the end of the song before the next one starts playing (just a short burtst of silence). I suspect this may be related to the transition or buffer.

I was thinking to add the following line below to the Custom Configuration:

```
<limits>
  <queue-size>2048000</queue-size>
  <burst-size>128000</burst-size>
</limits>

```
Could anyone tell me if someone has tried this out and if this will fix my problem?

Thank you for your assistance!

ps: this does not happen to every song. Let's say 40% of the songs.

**Comment by vitoyucepi** [comment]
Hi @heathcliff2019,
I'm unsure about the reason for your issue, but the code snippet you're trying to paste will definitely not work. Your code is for Icecast, not Liquidsoap.

To solve the problem, I'd like you to share
1. System information like OS version, hardware, etc.
2. Installation method.
3. Azuracast version.
4. Liqiudsoap logs with the undesired behavior.

Please notice that Azuracast developers may provide a better solution because they have better knowledge of the application-specific processes.

**Comment by RM-FM** [comment]
Hello @heathcliff2019,

I believe the Azuracast GitHub may be a better place for your question. You didn’t mention whether you added manual cue/fade values to your tracks or if you're using Autocue. Please keep in mind that without using one of these options, it's quite common for some songs to have silence at the end.

**Comment by heathcliff2019** [comment]
Hey @RM-FM, my problem is I think identical with https://github.com/AzuraCast/AzuraCast/issues/7386 but I am not using StereoTool. 

Since this is Liquidsoap I can't apply buffer=2.5, prebuffer=1.5.  My Memory & CPU Load are all good so that can be ruled out too. 

So I fine-tuned settings and I will come back next week.

This is about a new installation (one-click installation provided by Azuracast)

**Comment by heathcliff2019** [comment]
As promised I come back on this and so far so good.

What did I changed?

[unchanged] = Method has not been changed.

1. AutoDJ Queue Length: from 10 to 20.
2. Liquidsoap Performance Tuning: from disabled to balanced.
3. Duplicate Prevention Time Range (Minutes): from standard value to 240.
4. Enable AutoCue (Beta): Enabled  [unchanged].
5. Audio Post-processing Method: disabled  [unchanged].
6. I replace album covers with a 72dpi, 250x250px, JPG (quality 70%) [unchanged], _but Azura is changing it back to 96DPI, which is unnecessary for online display and gives the system extra work for no good reason_.
7. I always remove special characters and replace spaces with _ (underscore) [unchanged]. _I use a script to do that before uploading_.

ps: I dropped the cover issue here https://github.com/AzuraCast/AzuraCast/discussions/7537

@RM-FM @vitoyucepi
